datumaro/datumaro/cli/commands/add.py

```diff
@@ -0,0 +1,8 @@
+
+# Copyright (C) 2019 Intel Corporation
```

Comment:
@zhiltsov-max , please update all copyright comments. It should have 2020 or 2019-2020 years